src/lineup/internal/LineUpPanelActions.ts

```diff
@@ -216,15 +217,15 @@ export default class LineUpPanelActions extends EventHandler {
     const node = this.node.ownerDocument.createElement('div');
     node.classList.add('btn-group', 'download-data-dropdown');
     node.innerHTML = `
-      <button type="button" class="dropdown-toggle fa fa-download" style="width: 100%;" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Download Data">
+      <button type="button" class="dropdown-toggle fa fa-download" style="width: 100%;" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="${i18n.t('tdp:core.lineup.LineupPanelActions.downloadData')}">
       </button>
       <ul class="dropdown-menu dropdown-menu-${this.isTopMode ? 'left' : 'right'}">
-        <li class="dropdown-header">Download All Rows</li>
-        <li><a href="#" data-s="a" data-t="xlsx">Microsoft Excel (xlsx)</a></li>
-        <li class="dropdown-header" data-num-selected-rows="0">Download Selected Rows Only</li>
-        <li><a href="#" data-s="s" data-t="xlsx">Microsoft Excel (xlsx)</a></li>
+        <li class="dropdown-header">${i18n.t('tdp:core.lineup.LineupPanelActions.downloadAll')}</li>
+        <li><a href="#" data-s="a" data-t="xlsx">${i18n.t('tdp:core.lineup.LineupPanelActions.excel')}</a></li>
+        <li class="dropdown-header" data-num-selected-rows="0">${i18n.t('tdp:core.lineup.LineupPanelActions.downloadSelectedRows')}</li>
+        <li><a href="#" data-s="s" data-t="xlsx">${i18n.t('tdp:core.lineup.LineupPanelActions.excel')}</a></li>
         <li role="separator" class="divider"></li>
-        <li><a href="#" data-s="s" data-t="custom">Customize &hellip;</a></li>
+        <li><a href="#" data-s="s" data-t="custom">${i18n.t('tdp:core.lineup.LineupPanelActions.customize')} &hellip;</a></li>
```

Comment:
Please add the `&hellip;` to the language string as some customizations might want to change this...